types/genesis_test.go

```diff
@@ -34,9 +34,11 @@ func TestGenesisGood(t *testing.T) {
 	assert.NoError(t, err, "expected no error for good genDoc json")
 
 	// create a base gendoc from struct
+	pubKey, err := crypto.GenPrivKeyEd25519().PubKey()
+	assert.Nil(t, err)
```

Comment:
`require` probably makes more sense because further execution is pointless if pubkey is nil, right?